actors/miner/src/lib.rs

```diff
@@ -4854,7 +4939,9 @@ impl ActorCode for Actor {
         Constructor => constructor,
         ControlAddresses => control_addresses,
         ChangeWorkerAddress => change_worker_address,
+        ChangeWorkerAddressExported => change_worker_address,
```

Comment:
the match or syntax is way better, i.e `ChangeWorkerAddress | ChangeWorkerAddressExported`

Reply:
Unfortunately, the macro as written doesn't support the `|` syntax

Reply:
Lets file an issue for that

Reply:
Yes! https://github.com/filecoin-project/builtin-actors/issues/1018